jwst/assign_wcs/util.py

```diff
@@ -572,10 +572,10 @@ def _bbox_from_shape(model):
 
     s_region = (
         "POLYGON ICRS "
-        " {0} {1}"
-        " {2} {3}"
-        " {4} {5}"
-        " {6} {7}".format(*footprint.flatten()))
+        " {0:.9f} {1:.9f}"
```

Comment:
This looks fine, but as an alternative you could have used the `round` function, e.g.

    round(math.pi, 9)
    3.141592654

Reply:
`round()` is built into the string formatting sublanguage.  I had initially implemented it with a call to `np.round()` but then realized it was not necessary.  A nice discovery!